notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/NotificationHubInstallationHelper.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+class NotificationHubInstallationHelper {
```

Comment:
I hadn't actually meant a separate class, I just meant static helper methods in the class. Not worth holding up the PR though. I'll fix this up separately.